contrib/authors_update.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# See file LICENSE for terms.
+#
+
+set -eu
+set -o pipefail
```

Comment:
can marge into `set -eEx -o pipefail`

Reply:
fixed, but i'd prefer keep -x for CI only.

Reply:
IMO should add `E`

Reply:
fixed